src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/CallGCSuppressFinalizeCorrectlyTests.cs

```diff
@@ -135,6 +136,47 @@ End Sub
             await VerifyVB.VerifyAnalyzerAsync(code);
         }
 
+        [Fact]
+        public async Task AsyncDisposableWithFinalizer_CSharp_NoDiagnostic()
```

Comment:
What about doing the same test for vbnet?

Reply:
Sure.

Reply:
@Evangelink, ValueTask doesn't seem to be supported in vb.net


https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask?view=netcore-3.1#remarks

> The use of the ValueTask type is supported starting with C# 7.0 and is not supported by any version of Visual Basic.